helm/values.yaml

```diff
@@ -0,0 +1,62 @@
+# Default values for koordinator.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+global:
+    pullSecretName: regcred
+    clusterSpecificDnszone: "496a8aa9ae8c48329209.northeurope.aksapp.io"
+    loginSpecificDnszone: "login.test.com"
+    appsSpecificDnszone: "app.test.com"
+    idpSpecificDnszone: "idp.test.com"
+    useLocalhost: false
+    traceEnabled: true
+    prometheusCollectEnabled: false
+
+strategy:
+    rollingUpdate:
+        maxSurge: 1
+        maxUnavailable: 1
+
+resources:
+    large:
+        cpu:
+            limit: 1
+            request: 35m
+        memory:
+            limit: 1Gi
+            request: 200Mi
+    medium:
+        cpu:
+            limit: 0.8
+            request: 35m
+        memory:
+            limit: 600Mi
+            request: 100Mi
+    small:
+        cpu:
+            limit: 0.5
+            request: 5m
+        memory:
+            limit: 200Mi
+            request: 20Mi
+
+vivwallet:
+    name: ViV-Wallet
+    image: "xcomponent/x4b-login:1.3"
```

Comment:
x4b-login?

Reply:
Perhaps we can replace login by vivwallet in every file lol